packages/block-library/src/image/block.json

```diff
@@ -62,6 +62,9 @@
 			"source": "attribute",
 			"selector": "figure > a",
 			"attribute": "target"
+		},
+		"fullScreen": {
```

Comment:
Let's try adding this to the block-alignment component itself so that it's available to all blocks that use wide alignments.

Reply:
> Let's try adding this to the block-alignment component itself so that it's available to all blocks that use wide alignments.

When read it I'd already done a different approach which is not aligned what your note above Sorry about this :-/. I've added a `Why adding a new <Toolbar />?` section in the [PR description](https://github.com/WordPress/gutenberg/pull/16738).